packet/packet.go

```diff
@@ -14,6 +14,8 @@ func Unmarshal(b []byte) (p ArtNetPacket, err error) {
 		return
 	}
 
+	notImplErr := fmt.Errorf("unimplemented opcode %#v found", h.OpCode)
```

Comment:
Please follow `errNotImpl` format and move error to package level using `errors.New()`, and wrap the error using `fmt.Errorf("%w")` only when needed/used.